fabfile.py

```diff
@@ -287,6 +287,8 @@ def pull_database_from_platform(c, environment_name):
     )
 
     try:
+        print("Renaming app...")
+        run_management_command(c, "rename_app insights stories", check_returncode=True)
```

Comment:
```suggestion
        # TODO: Rename command to be removed once applied in all local environments
        print("Renaming app...")
        run_management_command(c, "rename_app insights stories", check_returncode=True)
```

Reply:
I like this, although we have a PR that removes this bit. Its a good practice to have that comment.
Good eye :) :) 
Thanks @ababic

Reply:
Done.